test/functional/plugin/shada_spec.lua

```diff
@@ -2318,15 +2317,17 @@ describe('In plugin/shada.vim', function()
         '  + l            line number  2',
         '  + c            column       -200',
       })
-      nvim_command('1,3w ' .. fname .. '.tst')
-      nvim_command('1,3w ' .. fname)
-      nvim_command('1,3w ' .. fname_tmp)
+      local write_cmd = '1,3w'
+      if helpers.iswin() then write_cmd = write_cmd .. '!' end
```

Comment:
What's going on here? If this is a bug let's not work around it. Otherwise just use "!" for both unix and win.

Reply:
I don't know myself. I assume nvim is not notified when a file is deleted. I always get `E13` on all Appveyor builds if I remove this line.